dali/sasha/saxref.cpp

```diff
@@ -803,7 +803,12 @@ class CNewXRefManager: public CNewXRefManagerBase
         rawgrp->queryNode(0).endpoint().getHostText(gname.clear());
         clusters.append(gname.str());
         clusterscsl.append(',').append(gname.str());
-        if (basedir.length()==0) {
+        if (isContainerized())
+        {
+            rootdir.set(getStoragePlane(_clustname)->queryProp("@prefix"));
+            assert(!rootdir.isEmpty());
+        }
+        else if (basedir.length()==0) {
```

Comment:
trivial/style: mixture of K&R vs Allman style. Best to keep it consistent with existing function style or change them all within a function, and for new code Allman style.